public/locales/es/administration.json

```diff
@@ -179,6 +181,10 @@
             "oct": "oct",
             "nov": "nov",
             "dec": "dic"
-        }
+        },
+        "downloadAll": "Descargar datos de todos los sensores",
+        "whichSensor": "¿Para qué sensor le gustaría descargar datos?",
```

Comment:
Should it be qué or cuál since they are choosing from a specific list of options